feature-utils/poly-look/babel.config.js

```diff
@@ -1,3 +1,8 @@
 module.exports = {
   presets: ["@babel/preset-env", "@babel/preset-react"],
+  env: {
+    test: {
+      presets: [["@babel/preset-env", { targets: { node: "current" } }]],
```

Comment:
Do you need to repeat this preset?

Reply:
Yes we had an error otherwise that could just be fixed by this I found out after a lot of research

Reply:
The thing is that what this seems to do is to change the default targets, I guess. So I'm not totally sure about what it solves. What was the error?

Reply:
Remove it and find out :D 
Here: 
` ReferenceError: regeneratorRuntime is not defined`

Reply:
https://github.com/facebook/jest/issues/3126

Reply:
But [this comment](https://github.com/facebook/jest/issues/3126#issuecomment-520061462) eliminates the first preset, instead of adding two of them.
At any rate, if it's fixed, well, who's to say anything, but I'd like to know why that happens and what's the origin, so that we know how to avoid happening again. Neither that issue nor [this SO answer](https://stackoverflow.com/questions/33527653/babel-6-regeneratorruntime-is-not-defined) provide a lot of insight, except something related to polyfills or somesuch. It would be nice to know the precise code that does that, too.

Reply:
Yeah I agree yet we searched for hours and looked at many threads. No real answers. At that point I was happy with it being fixed

Reply:
Nah, you need both. One default, the other for tests... Well, never mind. Let's just leave this conversation here just in case we bump into the same problem in the future.